docs/doc.nix

```diff
@@ -1,3 +1,5 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Docs are actually under different license - CC - but I'll add a commit to this branch to update this.

Reply:
Done with [414e807](https://github.com/tiiuae/ghaf/pull/77/commits/414e807e94e695e6ec21c7946f4f4bc2af250ced)